### ISSUE TITLE
Property should evaluate in addColumn > column > constraint

### ISSUE THREAD

**Issue by robinvandenbogaard** [liquibase/liquibase#1253]
## Environment
Wildfly 19
Updating multiple schemas in parallel using JBatch Processing
Java 11

**Liquibase Version**:
10.3

**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>
EJB

**Database Vendor & Version**:
Postgres 11

**Operating System Type & Version**:
Windows 10 Pro

## Description
Given a property named `otherSchema`
A changeset containing this element does not evaluate to the value in otherSchema but instead in resolves to the defaultSchema.
```
		<addColumn tableName="mytable">
			<column name="somerelation" type="bigint">
				<constraints nullable="false"
					referencedTableSchemaName="${otherSchema}"
					referencedTableName="othertable"
					referencedColumnNames="id"
					foreignKeyName="fk_somerelation_othertable" />
			</column>
		</addColumn>
```
The result is that it can't find the table in the defaultSchema and the query fails.

If I break the statement into multiple parts like so, the property evaluates like expected.
```
		<addColumn tableName="mytable">
			<column name="somerelation" type="bigint">
				<constraints nullable="false"/>
			</column>
		</addColumn>
		<addForeignKeyConstraint baseTableName="mytable"
				baseColumnNames="somerelation"
				referencedTableSchemaName="${otherSchema}"
				referencedTableName="othertable"
				referencedColumnNames="id"
				constraintName="fk_somerelation_othertable"/>
```
The result is that it **can** find the table in the otherSchema and the query **succeeds**.

**Comment by molivasdat** [comment]
Thanks @robinvandenbogaard for writing up this issue.  We will add it to the list of issues.  In the meantime, if you have a PR to suggest the code change, that would be great!

**Comment by fabrykowski** [comment]
This seems to be a duplicate of #3313

**Comment by filipelautert** [comment]
We're currently triaging our backlog to ensure we can focus on the issues that matter most. With a large number of open issues, it becomes difficult to give each one the attention it deserves.

This issue has been closed due to inactivity — there have been no updates for over 30 days. If this is still relevant to you, please feel free to reopen it or create a new issue with updated context. We appreciate your contribution!
